Tests/demistomock/demistomock.py

```diff
@@ -859,7 +859,15 @@ def createIncidents(incidents, lastRun=None, userID=None):
     Creates incident in long running execution
 
     Args:
-      incidents (list): List of incident objects to create
+      incidents (list):
+        List of incident objects to create, with the following required keys, and some common optional keys
+            - name (required) - str
+            - type (required) - if not provided will create unclassified incident) - str
```

Comment:
so not required?

Reply:
it's not required by the server, but we don't want to create incidents that are unclassified, so it's "required" 🙂

Reply:
```suggestion
            - type (required - if not provided will create unclassified incident) - str
```